examples/misc/test/cheatsheet/arguments_test.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:examples/cheatsheet/optional_named_params.dart';
+import 'package:examples/cheatsheet/optional_positional_args.dart'
+    as optional_args;
+import 'package:examples/cheatsheet/optional_positional_args2.dart'
+    as defaulted_args;
+import 'package:test/test.dart';
+
+void main() {
+  test('print_name', () {
+    expect(() {
+      printNameTest();
+    }, prints('Avinash Gupta \nPoshmeister Moneybuckets IV\n'));
```

Comment:
nit: I realize this is testing an already existing snippet, but this string might be too wacky / goofy: https://developers.google.com/style/tone

Reply:
That's fair, we don't want the examples to introduce unnecessary confusion or distraction. There's a few examples like this and I'm not sure of the best replacement at the moment, so how about we move this to a separate, larger issue to consider the voice/tone/global understanding of various examples. If not, do you have any short term suggestions for more understandable arguments I could switch these for.

Reply:
Sounds good, let's consider this separately https://github.com/dart-lang/site-www/issues/3485